app/assets/javascripts/app/views/publisher/getting_started_view.js

```diff
@@ -19,58 +19,58 @@ app.views.PublisherGettingStarted = Backbone.View.extend({
   // initiate all the popover message boxes
   show: function() {
     this._addPopover(this.el_first_msg, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: 30,
-      id: 'first_message_explain',
-      placement: 'right',
+      id: "first_message_explain",
+      placement: "right",
       html: true,
-      container: 'body'
+      container: "body"
     }, 600);
     this._addPopover(this.el_visibility, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: 10,
-      id: 'message_visibility_explain',
-      placement: 'bottom',
+      id: "message_visibility_explain",
+      placement: "bottom",
       html: true,
-      container: 'body'
+      container: "body"
     }, 1000);
     this._addPopover(this.el_stream, {
-      trigger: 'manual',
+      trigger: "manual",
       offset: -5,
-      id: 'stream_explain',
-      placement: 'left',
+      id: "stream_explain",
+      placement: "left",
       html: true,
-      container: 'body'
+      container: "body"
     }, 1400);
 
     // hide some popovers when a post is created
-    this.$('.button.creation').click(function() {
-      this.el_visibility.popover('hide');
-      this.el_first_msg.popover('hide');
+    this.$("#submit").click(function() {
+      this.el_visibility.popover("hide");
```

Comment:
Is there a purpose for having the el_ prefix to every instance variable in getting_started_view.js. Can I just remove it?